clean up episodes: (#494)
carpentries-incubator/hpc-intro: episodes/11-connecting.Rmd

results.

If you have ever opened the Windows Command Prompt or macOS Terminal, you have
seen a CLI. If you have already taken "The Carpentries" courses on the UNIX
Shell or Version Control, you have used the CLI on your *local machine*
extensively. The only leap to be made here is to open a CLI on a *remote
machine*, while taking some precautions so that other folks on the network
can't see (or change) the commands you're running or the results the remote

> machine sends back. We will use the *Secure Shell* protocol (or SSH) to open an

Reviewer: ```suggestion
machine sends back. We will use the _**S**_ecure _**Sh**_ell protocol (or SSH) to open an
```

Author: rather nested in form of _**S**_ecure _**Sh**_ell as it does not render properly.

I suggest using _**S**ecure **Sh**ell_.

Reviewer: I checked with AI, it lied! Suggestion is good

> machine sends back. We will use the _**S**ecure **Sh**ell_ protocol (or SSH) to open an

Reviewer: I think you mentioned this syntax doesn't work, but if it does then I think it is good.

set these arguments before clicking "connect." From the terminal, you'll write
something like `ssh userName@hostname`, where the argument is just like an

> email address: the "@" symbol is used to separate the personal ID from the

Reviewer: Let's be consistent with what is used in the next sentence
```suggestion
something like `ssh username@hostname`, where the argument is just like an
email address: the "@" symbol is used to separate the personal ID from the
```

address of the remote machine. In this lesson, `r config$remote$user` represents

> the _username_ on the remote system, and `r config$remote$login` represents

Reviewer: ```suggestion
the `userName` on the remote system, and `r config$remote$login` represents
```

the cluster's login address (_hostname_). Together, they form the SSH command:

> `r paste0("ssh ", config$remote$user, "@", config$remote$login)`.

Reviewer: I wouldn't provide the command as it typically will not work as shown (it usually needs a key or further arguments) 
```suggestion
the how to find the cluster on the internet (`hostname`).
```

set these arguments before clicking "connect." From the terminal, you'll write
something like `ssh userName@hostname`, where the argument is just like an
email address: the "@" symbol is used to separate the personal ID from the
address of the remote machine. In general, the `hostname` appears as

> `r config$remote$login` or sometimes just as `r config$remote$host`.

Reviewer: Not sure what you mean by `appears`. Do you mean when you execute the command, what then appears? Or that `hostname` is replaced with in the command? I'm assuming the latter: 

```suggestion
address of the remote machine. In our case, the `hostname` in the command is replaced by
`r config$remote$login` and `r config$remote$user` is our personal ID on that system.
```

Author: My point is on some machines, the hostname is used `hostname` and on some machines they use the login domain `hostname -d`.

Reviewer: Ah, I believe the `hostname` here is not the command `hostname` but a reference to the placeholder in the ssh command of line 58. That is perhaps confusing though

Author: `ssh userName@hostname`

On some clusters, the hostname is used as `hostname` on some machines and on few other machines it is `r config$remote$login`.

here `host is login1` or `host is cluster.hpc-carpentry.org`
As an example:
Ideally, the `ssh userName@hostname` is:
- `ssh userName@login1`, uses 'login*' or
- `ssh userName@cluster.hpc-carpentry.org`, uses 'the dns domain'

Reviewer: But `ssh userName@login1` will never succeed as a command as there's no domain. By getting into the details of what the command should look like here (which comes later anyway), I think we would be missing the point of the email analogy: your user @ some domain

Author: As per standard setups:

Basic SSH: `ssh username@cluster.hpc-carpentry.org` works with the full domain.

Additionally, we can configure SSH like this in `~/.ssh/config`:

```
Host login1
    User username
    IdentityFile ~/.ssh/ed25519_id
    HostName cluster.hpc-carpentry.org
```

We can simply run `ssh login1`.

In this case, `login1` is an alias defined in `~/.ssh/config` that maps to the cluster's hostname. It is therefore equivalent to `ssh username@cluster.hpc-carpentry.org`.

Without such configuration (or appropriate DNS/ host resolution), `login1` would not resolve as a usable hostname or SSH target on a typical external machine.

The [Lesson Setup](../learners/setup.md) provides instructions for installing
a shell application with SSH. If you have not done so already, please open that

> shell application with a Unix-like command line interface on your system.

Reviewer: I think this is marginal, I probably would say `to` is correct
